File: 100-million-calc/Swift/HundredMillionCalc.swift
import Foundation

let iterations = 100_000_000

print("計算回数: \(iterations)")

let startTime = DispatchTime.now()

var result = 0.0
for i in 0..<iterations {
    result += sin(Double(i)) * cos(Double(i))
}

print("計算結果: \(result)")

File: 100-million-calc/C/HundredMillionCalc.c
#include <stdio.h>
#include <math.h>

int main() {
    const long long iterations = 100000000;
    printf("計算回数: %lld\n", iterations);
    
    
    double result = 0.0;
    for (long long i = 0; i < iterations; i++) {
        result += sin((double)i) * cos((double)i);
    }
    
    printf("計算結果: %f\n", result);
    
    return 0;
}